### DISCUSSION TITLE
Longer directory history

### DISCUSSION THREAD

**Discussion by lorenzo-gi** [doublecmd/doublecmd#1273] [Suggestions]
Hello,

I tried [fman](https://fman.io/) and [FileSide](https://www.fileside.app) that both have a quick way to go back to visited folders by typing part of the path.
Then, I have discovered that doublecmd already does that and it is great: at least for me it allows for very quick jumps to frequently used folders (like the Hot Directory list but filled automatically instead of by hand).
Then I discovred that doublecmd only saves the last 50 paths and I think this hinders the funcionality. 
Is it already possible to set the history length to more than 50? 
The two programs above probably also rate the folders based on how often you visit them, but in my opinion this is less important.

**Comment by alexx2000** [comment]
Hi, no.

**Comment by lorenzo-gi** [comment]
Ok, then I think having the lenght of the history as a configuration would really improve this kind of usage. In the forum I have already seen a couple of other people wondering about longer history.

**Reply by kongomongo** [reply]
I absolutely second this, it would make the history immensely more powerful! I would put at least 500 there ;-)

**Comment by lorenzo-gi** [comment]
I think giving the possibility to change the size would be a nice benefit
without any downside

Il gio 29 feb 2024, 09:54 kongomongo ***@***.***> ha scritto:

**Comment by kongomongo** [comment]
trying to get this implemented: https://github.com/doublecmd/doublecmd/pull/1710
